pio-proc/Cargo.toml

```diff
@@ -18,8 +18,8 @@ syn = "1.0"
 quote = "1.0"
 codespan-reporting = "0.11"
 pio = { path = "..", version = "0.2.0" }
-pio-parser = { path = "../pio-parser", version = "0.2.0" }
+pio-parser = { path = "../pio-parser", version = "0.2.1" }
```

Comment:
Is there a reason for updating this dependency? By default, cargo would select the latest available version anyway, so best case this doesn't change anything. But it would break builds where some other crate depends on pio-parser with `version = "=0.2.0"`. I have no idea why anyone would do that here, but in general, I prefer specifying the _oldest_ working patch release of a dependency.

Of course, this change is correct if for some reason pio-proc can no longer work with pio-parser 0.2.0.

Reply:
There was no particular reason for this update. It just struck me as odd that it didn't depend on the newest version, but your reasoning makes sense.